src/Model/AbstractArticle.php

```diff
@@ -354,7 +354,7 @@ public function getSubtitle()
     /**
      * {@inheritdoc}
      */
-    public function setTags(array $tags = null)
+    public function setTags(iterable $tags = null)
```

Comment:
This is a BC-break, as demonstrated here: https://3v4l.org/7d5Xh, please remove this change.

Reply:
we still didn't release this bundle at all, so it is valid.